pkg/plugins/resources/postgres/pgx_store.go

```diff
@@ -465,3 +518,7 @@ func registerMetrics(metrics core_metrics.Metrics, pool *pgxpool.Pool, poolName
 	}
 	return nil
 }
+
+func (r *pgxResourceStore) Begin(ctx context.Context) (store.Transaction, error) {
+	return r.pool.Begin(ctx)
```

Comment:
IIUC the default IsoLevel is `read committed`, but the only thing we're trying to achieve here is batching – executing several operation on the same acquired connection. So it's probably safe to lower the IsoLevel to `read uncommitted`.

Reply:
I cannot find proof that it improves perf 🤔 Do you have sources for this?

Reply:
Normally that'd increase concurrency, but I see there is no difference in PostgreSQL 😞 

https://www.postgresql.org/docs/current/transaction-iso.html

> In PostgreSQL, you can request any of the four standard transaction isolation levels, but internally only three distinct isolation levels are implemented, i.e., PostgreSQL's Read Uncommitted mode behaves like Read Committed. This is because it is the only sensible way to map the standard isolation levels to PostgreSQL's multiversion concurrency control architecture.